tmuxp/cli.py

```diff
@@ -1185,10 +1185,13 @@ def command_convert(confirmed, config):
     """Convert a tmuxp config between JSON and YAML."""
 
     _, ext = os.path.splitext(config)
-    if 'json' in ext:
+    ext = ext.lower()
+    if ext == '.json':
         to_filetype = 'yaml'
-    elif 'yaml' in ext:
+    elif ext in ['.yaml', '.yml']:
```

Comment:
This looks good!

Can we have a test for it?  (e.g. in https://github.com/tmux-python/tmuxp/blob/master/tests/test_cli.py#L725-L733)

Reply:
@catroll at your request I can also merge this and I can write the test after merge. Whichever works best for you :)

Reply:
Should I write some test first?
But I'm not good enough at pytest, so maybe need some time to learn it.

Reply:
I will give you the choice, since it's an opportunity to learn.  I can wait.

If you'd like to learn and write it yourself, the instructions here:
- https://tmuxp.git-pull.com/developing.html should let you set it up.  
- [pytest documentation](https://docs.pytest.org/en/6.2.x/contents.html)

As an alternative, if that's too much effort for a fix of this size, please let me know if you'd like to skip writing a test this time.

Reply:
> 

OK, I will write test myself, after work.

Reply:
i write some tests
please review it
is it ok?
@tony

Reply:
@catroll  

I ran the tests, I will take a look evening time (by me)

P.S. To avoid merge commits, you can do real rebase, e.g.

In your branch:

`git remote add tmux-python https://github.com/tmux-python/tmuxp.git`
`git pull --rebase tmux-python master`
`git push --force-with-lease`

Reply:
ok